invenio_drafts_resources/resources/records/resource.py

```diff
@@ -28,3 +34,49 @@ def create(self):
         item = self.service.create(
             g.identity, data, links_config=self.config.draft_links_config)
         return item.to_dict(), 201
+
+
+class RecordVersionsResource(CollectionResource, ConfigLoaderMixin):
+    """Record versions resource."""
+
+    default_config = RecordVersionsResourceConfig
+
+    def __init__(self, service=None, config=None):
+        """Constructor."""
+        super(RecordVersionsResource, self).__init__(
+            config=self.load_config(config))
+        self.service = service or RecordDraftService()
+
+    def _get_es_preference(self):
```

Comment:
Is this function not repeated from somewhere else? perhaps just extract the function and reuse it both places.

Reply:
I thought to create a `BaseRecordResource` here https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/resources/records/resource.py#L23 . Should I go for it?